src/main/java/com/tileman/TilemanModePlugin.java

```diff
@@ -151,22 +150,10 @@ public void onGameTick(GameTick tick) {
             return;
         }
 
-        // This number will change should any config that affects the counter be changed
-        int currentConfigState = config.tilesOffset()
-                + config.warningLimit() * 10
-                * (config.includeTotalLevel() ? -1 : 1)
-                * (config.excludeExp() ? -1 : 1);
-        int currentTotalTiles = (int) client.getOverallExperience() / 1000;
-
         if (lastTile == null || (lastTile.distanceTo(playerPosLocal) != 0)) {
             handleMovement(playerPosLocal);
             updateTileCounter();
-        } else if (currentConfigState != configState || totalTiles != currentTotalTiles) {
```

Comment:
Don't we still need this for total tiles?

Reply:
Shouldnt need it I dont think. 

The lines above handle movement and the only other way total tiles would be changed is config. 

Any actions that add/remove tiles call the update method too